src/main/scala/cromwell/engine/ExecutionTable.scala

```diff
@@ -0,0 +1,11 @@
+package cromwell.engine
+
+import cromwell.binding.{Call, WdlBinding}
+
+/**
+ * Corresponds to the "execution table" of our discussions.
+ */
```

Comment:
Agreed.  I originally called these `ExecutionState` and `SymbolState`, but we kept using the "table" terms in conversation.  If we can agree on what we'd like to call these I'm more than happy to change their names.